docs/atomic-uninstall.1.md

```diff
@@ -49,5 +50,8 @@ Any additional arguments will be appended to the command.
 **-n** **--name**=""
    If name is specified `atomic uninstall` will uninstall the named container from the system, otherwise it will uninstall the container images.
 
+**--system**
+   Uninstall a system container.
```

Comment:
I think we should do our best to prevent duplicate named containers.  As  you state we could end up with bad results when we use alternate names.  

If I uninstall a --system container, I would end up with a docker container which I could later uninstall.

If we prevent the installing of a --system container if a docker container exists, then we don't end up in this state unless a user went behind our back.  I think this is better then figuring out what happens if a user tries to remove a system container without specifying --system.